web/html/src/build/webpack.config.js

```diff
@@ -3,6 +3,7 @@ const LicenseCheckerWebpackPlugin = require("license-checker-webpack-plugin");
 const {pages} = require("../manager/index");
 const webpack = require("webpack");
 const CleanWebpackPlugin = require('clean-webpack-plugin');
+const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
```

Comment:
do we still need this?

Reply:
Ok, I'll remove it. Can be added back at development time if needed.